From 067b9ff948d68e3e18e6db563a2f14d9d14c03da Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@debian.org>
Date: Thu, 2 Jan 2014 13:13:12 +0000
Subject: Unset IFS at nroff startup

It's questionable whether we should forward this; as noted in the bug
trail, there's a decent argument that this is a dash bug.

Bug-Debian: https://bugs.debian.org/541621
Forwarded: no
Last-Update: 2018-03-05

Patch-Name: nroff-ifs.patch
---
 src/roff/nroff/nroff.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/roff/nroff/nroff.sh b/src/roff/nroff/nroff.sh
index 523c4531..7ca26b10 100644
--- a/src/roff/nroff/nroff.sh
+++ b/src/roff/nroff/nroff.sh
@@ -20,6 +20,8 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+unset IFS
+
 prog="$0"
 
 # Default device.
